rules/no-skip-test.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+var util = require('../util');
+var createAvaRule = require('../create-ava-rule');
+
+module.exports = function (context) {
+	var ava = createAvaRule();
+	return ava.merge({
+		MemberExpression: function (node) {
+			if (ava.isTestFile && node.property.name === 'skip' && util.isCallFromTestObject(node)) {
```

Comment:
This is not wrong since @jfmengels focused on `MemberExpression` then traverse it up.

However if you want to use existing properties and methods, this could be written as follows.
```js
	return ava.merge({
		CallExpression: function (node) {
			if (ava.isTestFile && ava.currentTestNode === node && ava.hasTestModifier('skip')) {
				context.report(node, 'No tests should be skipped.');
			}
		}
	});
```

`ava.currentTestNode === node` means "on entering `test` method". Fires exactly once for each `test` method.